js/hotgraphicModel.js

```diff
@@ -0,0 +1,15 @@
+import ItemsComponentModel from 'core/js/models/itemsComponentModel';
+import tooltips from 'core/js/tooltips';
+
+export default class HotgraphicModel extends ItemsComponentModel {
+
+  setUpItems() {
+    super.setUpItems();
+    this.getChildren().forEach(child => {
+      if (!child.get('_tooltip')) return;
+
+      tooltips.register(child.get('_tooltip'));
+    })
```

Comment:
Automatic `_id` generation reduces schema to just `_isEnabled` and `text`.
```js
    const id = this.get('_id');
    this.getChildren().forEach((child, index) => {
      const tooltip = child.get('_tooltip');
      if (!tooltip?._isEnabled) return;
      tooltip._id = `hotgraphic-pin-${id}-${index}`;
      tooltips.register({
        _classes: [ 'hotgraphic__tooltip' ],
        ...tooltip
      });
    });
```
I would move the `_isEnabled` check here and remove it from the template.

Note: Need to add the schema and example.json and readme